qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -15,13 +15,9 @@
     setsebool_cmd = "setsebool virt_qemu_ga_read_nonsecurity_files %s"
     getsebool_cmd = "getsebool -a | grep virt_qemu_ga_read_nonsecurity_files |awk '{print$3}'"
     backup_file = /etc/sysconfig/qemu-ga-bk
-    black_list_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
+    black_file_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
     recovery_black_list = mv -f ${backup_file} /etc/sysconfig/qemu-ga
-    black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
-    black_list_spec = BLOCK_RPCS
```

Comment:
We can add these two parameter to cfg.

black_list_spec = "BLACKLIST_RPC"
black_list_check_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"

Reply:
maybe the `black_list_spec = "BLACKLIST_RPC"` can be added, 
`black_list_check_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"`  you mean `black_list_change_cmd` right?  this is not the same as the following you ask me to operate. those are different commands.

Reply:
@leidwang done. 
RESULTS    : PASS 21 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0